From 3803c37aff7b1fc6412588c99c1560c549aad00c Mon Sep 17 00:00:00 2001
From: Angus Thomas <athomas@redhat.com>
Date: Mon, 6 Oct 2014 14:51:51 +0100
Subject: [PATCH] Remove pbr dependency

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 782bb21..6357e08 100644
--- a/setup.py
+++ b/setup.py
@@ -25,5 +25,4 @@ except ImportError:
     pass
 
 setuptools.setup(
-    setup_requires=['pbr>=1.8'],
     pbr=True)
